.github/scripts/close_nonexistent_disable_issues.py

```diff
@@ -0,0 +1,174 @@
+import argparse
+import json
+import multiprocessing as mp
+import os
+import re
+import tempfile
+from typing import Any, Dict, List, Optional, Tuple
+
+import requests
+import rockset  # type: ignore[import]
+
+LOGS_QUERY = """
+with
+    shas as (
+        SELECT
+            push.head_commit.id as sha,
+        FROM
+            commons.push
+        WHERE
+            push.ref = 'refs/heads/viable/strict'
+            AND push.repository.full_name = 'pytorch/pytorch'
+        ORDER BY
+            push._event_time DESC
+        LIMIT
+            5
+    )
+select
+    id,
+    name
+from
+    workflow_job j
+    join shas on shas.sha = j.head_sha
+where
+    j.name like '% / test%'
+    and j.name not like '%rerun_disabled_tests%'
+    and j.name not like '%mem_leak_check%'
+"""
+
+TEST_EXISTS_QUERY = """
+select
+    count(*) as c
+from
+    test_run_s3
```

Comment:
If this collection is too slow/expensive to query, a good approximation is `https://console.rockset.com/collections/details/commons.rerun_disabled_tests`.  IMO, having no mention in any of the jobs + having no record in `commons.rerun_disabled_tests` in the past N days (this could be any value like a month) should also work

Reply:
Thankfully, I think count is pretty fast.  I'm also super conservative with this script, I'm worried that there might be some weirdness for reruns related to manually added skips in the code and also platform differences.